src/handler/mongoHandler.js

```diff
@@ -61,20 +61,13 @@ class MongoHandler {
     const options = {upsert: true};
     coursedb.updateOne(searchQuery, updateQuery, options)
       .then((response) => {
-        const body = {
-            uuid: content.uuid,
-            action: content.action
-        };
         if (response.upsertedCount === 0) {
           this.logger.info(`[ ${logTag} ] wish ${content.name} by ${content.wisher} already exists`);
-          body.statusCode = 409;
-          body.message = "Wish by that name and author already exists. Please try another.";
+          resolve(false);
```

Comment:
This will requeue the create.

Reply:
Should it just be resolve(true) then? Do we just assume the data is lost?

Reply:
If it is false, it would just be requeued. Maybe we should make wish create also synchronous.